lib/buildingsync/model_articulation/loads_system.rb

```diff
@@ -148,60 +150,12 @@ def get_building_section(building_sections, standard_building_type, standard_spa
 
     def adjust_people_schedule(space_type, building_section, model)
```

Comment:
The implementation of the `adjust_people_schedule` function uses the `get_building_section` function, which only returns a single building_section:
https://github.com/BuildingSync/BuildingSync-gem/blob/865aaa7164003d753a6e77c7838afffdbd1e4637/lib/buildingsync/model_articulation/loads_system.rb#L77

Please describe how this should / does work for multiple sections with different typical occupant hours.

Reply:
We iterate over all space types and adjust the related schedules in each schedule set of each space type. 
The building section is needed to get the parameter hours_per_week that is defined at that level.